governance/src/main/java/org/apache/servicecomb/governance/utils/CustomMatch.java

```diff
@@ -22,5 +22,6 @@
 public interface CustomMatch {
     String errorMessageForNotImplements = " didn't implement interface org.apache.servicecomb.governance.utils.CustomMatch";
     String errorMessageForAbstractClass = " should be a instantiable class rather than abstract class or other else";
+    String infoMessageForCreatingClass = "is not in spring container, create one and register it to spring container";
```

Comment:
If we don't let the user to use this constant string, it's better just put them into the log method.